scene/gui/text_edit.cpp

```diff
@@ -4000,6 +4000,11 @@ Point2i TextEdit::search(const String &p_key, uint32_t p_search_flags, int p_fro
 		int last_pos = -1;
 
 		while (true) {
+			if (p_search_flags & SEARCH_IGNORE_COMMENTS) {
+				if (text_line.begins_with("#")) {
```

Comment:
The same as my other comment regarding `\t`

Reply:
Debug dump
```
text_line = {String} U"\t# clicked"
 _cowdata = {CowData<char32_t>} {_ptr=0x0000026460c04320 U"\t# clicked"}
```

Reply:
hmm need to find a way to remove \t's